cypress/integration/Modal.spec.ts

```diff
@@ -140,9 +151,7 @@ describe('Modal', () => {
 
         context('when the overlay is clicked', () => {
           beforeEach(() => {
-            cy.findByLabelText('Delete Item')
-              .pipe(h.modal.getOverlay)
-              .click('top');
+            cy.get('body').click('top');
```

Comment:
We really only care about clicking outside the modal. I don't think we need to encode anything about the Modal outside that

Reply:
Should we update the `'when the overlay is clicked'` context descriptions to `'when clicking outside the modal'` (or something similar) then?

Reply:
Yes, thank you!